Plugins/DeleteUnusedBranches/CheckBoxHeaderCellEventArgs.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace DeleteUnusedBranches
+{
+    public class CheckBoxHeaderCellEventArgs : EventArgs
+    {
+        public bool Checked { get; set; }
```

Comment:
`EventArgs` payload (with few exceptions) must be readonly, otherwise you can't guarantee stability and consistency.

```
public class CheckBoxHeaderCellEventArgs : EventArgs
{
    public CheckBoxHeaderCellEventArgs(bool checked)
    {
        Checked = checked;
    }

    public bool Checked { get; }
}